API/API.mli

```diff
@@ -2160,10 +2160,14 @@ module Decl_kinds :
 sig
   type polymorphic = bool
   type cumulative_inductive_flag = bool
+
   type recursivity_kind = Declarations.recursivity_kind =
-    | Finite
-    | CoFinite
-    | BiFinite
+    | Finite (** = inductive *)
+        [@ocaml.deprecated "Please use [Declarations.Finite"]
+    | CoFinite (** = coinductive *)
+        [@ocaml.deprecated "Please use [Declarations.CoFinite"]
+    | BiFinite (** = non-recursive, like in "Record" definitions *)
```

Comment:
These comments should added to the non deprecated type.

Reply:
Updated. Anyways I hope the kernel part of API is going away soon.